src/main/java/bio/terra/service/dataset/DatasetDao.java

```diff
@@ -252,7 +267,7 @@ protected String getExclusiveLock(UUID id) {
      * @param id the dataset id
      * @return the array of flight ids that hold shared locks. empty if no shared locks are taken out.
      */
-    protected String[] getSharedLocks(UUID id) {
+    public String[] getSharedLocks(UUID id) {
```

Comment:
Does this need to be made public? I think if you call it from the test/dataset directory, where most of the dataset-related connected/integration tests are anyway, then we don't need to leak this method outside the package. This method was originally made protected so that it can only be used by tests and not by any other API code.

Reply:
I was confused about this. I couldn't seem to directly call this protected method in test/../common/fixtures/ConnectedOperations.java without making it public. I'm guessing it has to do with file structure? 

New solution: I created a datasetDaoUtils class here: test/.../service/dataset/DatasetDaoUtils.java, and then I call this utility method in ConnectedOperations. Totally not tied to this solution, so let me know if you have another idea!